src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/BsdNetworkInterface.cs

```diff
@@ -128,7 +128,7 @@ public static unsafe NetworkInterface[] GetBsdNetworkInterfaces()
         /// <returns>The cached or new BsdNetworkInterface with the given name.</returns>
         private static BsdNetworkInterface GetOrCreate(Dictionary<string, BsdNetworkInterface> interfaces, string name, int index)
         {
-            BsdNetworkInterface oni;
+            BsdNetworkInterface? oni;
```

Comment:
Unrelated to your change, but what does _oni_ stand for here? 🤔

Reply:
might intended to name `bni` 😄